rma_sale/controllers/sale_portal.py

```diff
@@ -29,6 +29,8 @@ def request_rma(self, order_id, access_token=None, **post):
         mapped_vals = {}
         for name, value in post.items():
             row, field_name = name.split("-", 1)
+            if field_name.endswith('_id'):
```

Comment:
This is a very weak condition. It's your duty to check in your customizations if it's a string. All portal works this way. But it's as simple as always do `int(whatever)`, being already an integer or not.

Reply:
I know condition is weak, and could be improved by using a better check [1]. I can do it, if necessary. However, the problem is that every customization will receive faulty data anyway, because the values herein set will be directly passed to a `create` method. The only way a customization could check if the values are correct, is to override the `create` itself, but it would be highly expensive in terms of performance, I believe.

[1] i.e.: `if self.env['rma]._fields[field_name].type == 'many2one':`

Reply:
How the rest of the portal works with this? I don't think it's expensive to envelop the value with `int()`

Reply:
> How the rest of the portal works with this?

I'm sorry, I think I don't understand what you mean.

> I don't think it's expensive to envelop the value with int()

But overriding every model's `create` to check if every value is correct is more expensive than checking beforehand :)